tools/python/influx-migration/README.md

```diff
@@ -146,15 +146,35 @@ After meeting the prerequisites:
 
 2. **Provide Credentials**: Provide host addresses and ports as CLI options.
 
-3. **Verify Data**: Ensure the data is correctly transferred by:
+3. **Verify Data**: To manually verify data migration run the following commands before, on the source instance, and after migrating, on the destination instance:
 
-    a. Using the InfluxDB UI and inspecting buckets.
+    - **i**. List all buckets using the Influx CLI, where `<token>` is an operator token and `<host>` is the host of the source or destination instance, e.g., `https://<hostname>:8086` or `http://localhost:8086`.
 
-    b. Listing buckets with `influx bucket list -t <destination token> --host <destination host address> --skip-verify`.
+      ```shell
+      influx bucket list -t <token> --host <host>
```

Comment:
The example value for `--host` has been named `<host>` instead of `<endpoint>` since Timestream for InfluxDB refers to endpoints as the domain of an instance, i.e., a domain without a scheme or port.